src/FSLibrary/StellarKubeSpecs.fs

```diff
@@ -339,6 +339,7 @@ let WithProbes (container: V1Container) (probeTimeout: int) : V1Container =
             periodSeconds = System.Nullable<int>(1),
             failureThreshold = System.Nullable<int>(60),
             timeoutSeconds = System.Nullable<int>(probeTimeout),
+            initialDelaySeconds = System.Nullable<int>(180),
```

Comment:
Should we only set this delay when we are in a mode with pregenerated TXs?

Reply:
I was a bit surprised we didn't have initial startup delay configured actually, since we usually want to allow some time for core to startup before running the probe. I think it's worth keeping; 3 minutes is an overkill though, with the new version of this PR we should be creating a lot less transactions, so I changed it to 1 minutes. Let me know what you think (I pushed the change to 1 min)

Reply:
Yeah 1 minute sounds reasonable. My only concern is with very short lived tests. For example, would we potentially miss an error in `SimplePayment` with this, where the pod would fail before the health check probe was deployed?

Reply:
It shouldn't cause problems in tests, since if core is down, the mission wouldn't even be able to close ledgers. Liveness probe just helps detect failures and kill the containers. I did a bit of digging to figure out why we weren't using it: turns we used to, but removed it in https://github.com/stellar/supercluster/pull/8/files#diff-36aa58ec76c231cd7468617ddfd2a87e3cfd76f25dd3c278d0dcc5f71f184e00L274 in favor of startup probe. There were some issues with the startup probe, so we disabled it later in https://github.com/stellar/supercluster/pull/13, but forgot to bring the startup delay back. So I think 1 min here is fine (it's actually less than the previous default).